specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-02-01-preview/mfe.json

```diff
@@ -10461,6 +12958,18 @@
       },
       "additionalProperties": false
     },
+    "ComputeResource": {
```

Comment:
You add this model here, but other swagger files already contain model with the same name, and it has completely different properties.
This breaks the SDK generation and might produce ill-named SDKs. Please change the name of this model to something else.